modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -136,8 +140,291 @@ instead.
+			<Table.Cell backgroundColor='soap300'>
+				soap300
+			</Table.Cell>
+			<Table.Cell backgroundColor='soap400'>
+				soap400
+			</Table.Cell>
+		</Table.Row>
+		<Table.Row>
+			<Table.Cell>
+				TertiaryButton
+			</Table.Cell>
+			<Table.Cell/>
+			<Table.Cell>active</Table.Cell>
+			<Table.Cell>
+				color
+			</Table.Cell>
+			<Table.Cell color='frenchVanilla100' backgroundColor={cssVar(brand.primary.dark)}>
+				primary.dark
+			</Table.Cell>
+			<Table.Cell color='frenchVanilla100' backgroundColor={cssVar(brand.primary.darkest)}>
+				primary.darkest
+			</Table.Cell>
+		</Table.Row>
+		<Table.Row>
+			<Table.Cell>
+				TertiaryButton
+			</Table.Cell>
+			<Table.Cell/>
+			<Table.Cell>disabled</Table.Cell>
+			<Table.Cell>
+				borderColor
+			</Table.Cell>
+			<Table.Cell backgroundColor='frenchVanilla100'>
+				frenchVanilla100
+			</Table.Cell>
+			<Table.Cell backgroundColor='transparent'>
+				transparent
+			</Table.Cell>
+		</Table.Row>
+		<Table.Row>
+			<Table.Cell>
+				TertiaryButton (Icon Only)
+			</Table.Cell>
+			<Table.Cell/>
+			<Table.Cell>focus</Table.Cell>
+			<Table.Cell>
+				fill
+			</Table.Cell>
+			<Table.Cell color='frenchVanilla100' backgroundColor='blackPepper500'>
+				blackPepper500
+			</Table.Cell>
+			<Table.Cell color='frenchVanilla100' backgroundColor='blackPepper400'>
+				blackPepper400
+			</Table.Cell>
+		</Table.Row>
+		<Table.Row>
+			<Table.Cell>
+				TertiaryButton (Icon Only)
+			</Table.Cell>
+			<Table.Cell/>
+			<Table.Cell>hover</Table.Cell>
+			<Table.Cell>
+				fill
+			</Table.Cell>
+			<Table.Cell color='frenchVanilla100' backgroundColor='blackPepper500'>
+				blackPepper500
+			</Table.Cell>
+			<Table.Cell color='frenchVanilla100' backgroundColor='blackPepper400'>
+				blackPepper400
+			</Table.Cell>
+		</Table.Row>
+		<Table.Row>
+			<Table.Cell>
+				DeleteButton
+			</Table.Cell>
+			<Table.Cell/>
+			<Table.Cell>disabled</Table.Cell>
+			<Table.Cell>
+				backgroundColor
+			</Table.Cell>
+			<Table.Cell backgroundColor={cssVar(brand.error.light)}>
+				error.light
+			</Table.Cell>
+			<Table.Cell color='frenchVanilla100' backgroundColor={cssVar(brand.error.base)}>
+				error.base
+			</Table.Cell>
+		</Table.Row>
+		<Table.Row>
+			<Table.Cell>
+				TextInput
+			</Table.Cell>
+			<Table.Cell/>
+			<Table.Cell>disabled</Table.Cell>
+			<Table.Cell>
+				borderColor
+			</Table.Cell>
+			<Table.Cell backgroundColor='soap600'>
+				soap600
+			</Table.Cell>
+			<Table.Cell color='frenchVanilla100' backgroundColor='licorice100'>
+				licorice100
+			</Table.Cell>
+		</Table.Row>
+		<Table.Row>
+			<Table.Cell>
+				TextArea
+			</Table.Cell>
+			<Table.Cell/>
+			<Table.Cell>disabled</Table.Cell>
+			<Table.Cell>
+				borderColor
+			</Table.Cell>
+			<Table.Cell backgroundColor='soap600'>
+				soap600
+			</Table.Cell>
+			<Table.Cell color='frenchVanilla100' backgroundColor='licorice100'>
+				licorice100
+			</Table.Cell>
+		</Table.Row>
+  </Table.Body>
+</Table>
+
```

Comment:
Anything special you wanna call out here? The disabled delete button looks like the color is solid but we actually add an opacity now to the base color right? Might be worth adding as a note?

Reply:
Yeah definitely. I will address the opacity portion.